CoreUtilities/src/au/gov/asd/tac/constellation/utilities/graphics/FloatArray.java

```diff
@@ -134,6 +134,7 @@ public FloatArray(final int initialCapacity) {
             throw new IllegalArgumentException("Illegal Capacity: " + initialCapacity);
         }
         this.elementData = new float[initialCapacity];
+        size=0;
```

Comment:
Minor but it might be good to add some spaces here

Reply:
See latest changes, I think this is what you meant?

Reply:
Not quite. I meant `size=0` becoming `size = 0`